openapi.yaml

```diff
@@ -18172,6 +18178,18 @@ components:
         - JOB
         - HELM
         - ROUTER
+      title: ''
+    PublicServiceTypeEnum:
```

Comment:
Is it necessary to create a new enum? I'm not sure why we would like to keep `ROUTER` or where we use it